octobot_trading/exchange_data/ohlcv/channel/ohlcv_updater.py

```diff
@@ -32,6 +32,7 @@ class OHLCVUpdater(ohlcv_channel.OHLCVProducer):
     OHLCV_ON_ERROR_TIME = 5
     OHLCV_MIN_REFRESH_TIME = 1
     OHLCV_REFRESH_TIME_THRESHOLD = 1.5  # to prevent spamming at candle closing
+    OHLCV_REFRESH_RETRY_DELAY = 10
```

Comment:
Why an arbitrary delay ? `time_frame_sleep` should correspond to the next candle creation